src/game.cpp

```diff
@@ -7784,6 +7786,19 @@ bool pet_menu(monster *z)
         amenu.addentry(pheromone, true, 't', _("Tear out pheromone ball"));
     }
 
+    if (z->type->id == mtype_id("mon_dog") ||
+        z->type->id == mtype_id("mon_beakhound") || 
+        z->type->id == mtype_id("mon_beakhound2") || 
+        z->type->id == mtype_id("mon_beakhound3")) 
+    {
+        if (g->u.has_amount("syringe", 1)) {
+            amenu.addentry(inject, true, 'i', _("Inject mutagen"));
+        }
+        else {
+        amenu.addentry(inject, false, 'i', _("You need a syringe"));
```

Comment:
Needs an explanation for why would you need that syringe. "You need a syringe to apply serum" or something.